### ISSUE TITLE
Outdated Parse Server installs with bootstrap.sh

### ISSUE THREAD

**Issue by Rijak2005** [parse-community/parse-server#8513]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

The [getting started section](https://docs.parseplatform.org/parse-server/guide/#getting-started) of the parse server guide references the [bootstrap.sh](https://docs.parseplatform.org/parse-server/guide/#getting-started) script to install the parse server.
The problem is that this script installs an old version of the parse server (it installs 3.9.0) perhaps due to [this](https://github.com/parse-community/parse-server/blob/177891ea0eed54eff996d81749cfbfb32156ae87/bootstrap.sh#L166). So in order to use the latest version I had to upgrade it myself.
### Steps to reproduce

Follow the [getting started section](https://docs.parseplatform.org/parse-server/guide/#getting-started)
### Actual Outcome

Parse Server version 3.9.0 is installed
### Expected Outcome

Latest version of parse server should have been installed
### Environment

Server
- Parse Server version: `3.9.0`
- Operating system: `Debian 11`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `Local`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `6.0.5`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `local`

**Comment by mtrezza** [comment]
It installs Parse Server 3 and the current version is 6. If no-one has been complaining about this so far then maybe we should  remove the bootstrap script complete instead of maintaining it?

**Comment by Rijak2005** [comment]
Yeah you guys know better, it would be best to update the docs too in any case to help newbies like me to set up the Parse Server easily

**Comment by davidhakki** [comment]
What's funny is I accidentally did this recently and was scratching my head wondering why version 3 was installed. I assumed I messed something up.

**Comment by mtrezza** [comment]
Does anyone want to update the bootstrap script?

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [9.10.1-alpha.4](https://github.com/parse-community/parse-server/releases/tag/9.10.1-alpha.4)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [9.10.1](https://github.com/parse-community/parse-server/releases/tag/9.10.1)
